lib/extensions/PenguGuild.js

```diff
@@ -0,0 +1,20 @@
+const { Structures } = require("discord.js");
+
+module.exports = Structures.extend("Guild", Guild => {
+    /**
+	 * Pengu's Extended Guild
+	 * @extends {Guild}
+	 */
+    class PenguGuild extends Guild {
+
+        /**
+		 * @param {...Object} args Normal D.JS guild args
+		 */
+		constructor(...args) { // eslint-disable-line
```

Comment:
remove the disable line and fix indent

Reply:
Again the indent is just github since in the editor it looks fine